lib/tasks/report_inconsistent_facet_values.rake

```diff
@@ -0,0 +1,57 @@
+require "gds_api/rummager"
+require "json"
+
+class EmptyFacetValueError < StandardError
+end
+
+desc "Check for and report any non-expanded facet values"
+task :report_inconsistent_facet_values do
+  # List of facets to check
+  facets = %w{
```

Comment:
Hmm, could you add them to the `ALLOWED_FACET_THINGS` constant (and reuse that here)? 

They could have the same problem as the others.